rest/route/version_routes.go

```diff
@@ -75,6 +81,81 @@ func (vh *versionHandler) Execute(ctx context.Context, sc data.Connector) (Respo
 	}, nil
 }
 
+// activeVersionsHandler is a RequestHandler for fetchhing
+// all active versions for given projectId
+type activeVersionsHandler struct {
+	projectId string
+}
+
+func getActiveVersionsRouteManager(route string, version int) *RouteManager {
+	return &RouteManager{
+		Route: route,
+		Methods: []MethodHandler{
+			{
+				Authenticator:  &NoAuthAuthenticator{},
+				RequestHandler: &activeVersionsHandler{},
+				MethodType:     http.MethodGet,
+			},
+		},
+		Version: version,
+	}
+}
+
+// Handler returns a pointer to a new activeVersionsHandler.
+func (avh *activeVersionsHandler) Handler() RequestHandler {
+	return &activeVersionsHandler{}
+}
+
+// ParseAndValidate fetches the versionId from the http request.
+func (avh *activeVersionsHandler) ParseAndValidate(
```

Comment:
Have you run gofmt on this?

Reply:
Sure. It `gofmt` command is binded to save action in my editor. I am sure most lines should be less than 80 symbols and all lines must be shorter than 112 symbols (GitHub displays only this number of symbols in collapsed PR view).